fdm-core/src/db/schema.ts

```diff
@@ -39,7 +39,10 @@ export type farmsTypeInsert = typeof farms.$inferInsert
 export const acquiringMethodOptions = [
     { value: "nl_01", label: "Eigendom" },
     { value: "nl_02", label: "Reguliere pacht" },
-    { value: "nl_03", label: "In gebruik van een terreinbeherende organisatie" },
+    {
+        value: "nl_03",
+        label: "In gebruik van een terreinbeherende organisatie",
+    },
```

Comment:
use of  comma in the end before bracket is oke? not expecting a next object then?

Reply:
No, this follows a linter rule